release-content/0.15/migration-guides/15591_Text_rework.md

```diff
@@ -1,8 +1,28 @@
-TODO: very breaking
+The `Text` API in Bevy has been overhauled in several ways as part of Bevy 0.15.
+There are several major changes to consider:
 
-__Accessing text spans by index__
+- `ab_glyph` has been replaced with `cosmic-text`. These changes are mostly internal and the majority of users will not interact with either text backend directly.
+- each text section is now stored as a distinct entity within the standard hierarchy, rather than as a `Vec<TextSection>` on the `Text` component. Children of `Text`/`Text2d` entities with `Text` `Text2d` components will act as additional text sections.
```

Comment:
```suggestion
- each text section is now stored as a distinct entity within the standard hierarchy, rather than as a `Vec<TextSection>` on the `Text` component. Children of `Text`/`Text2d` entities with `TextSection` components will act as additional text sections.
```
Only root is different, sections are shared by all AFAIR

Reply:
The `TextSection` component doesn't exist.

Reply:
Right, the Component shared by both for sections is `TextSpan`.

Reply:
```suggestion
- each text section is now stored as a distinct entity within the standard hierarchy, rather than as a `Vec<TextSection>` on the `Text` component. Children of `Text`/`Text2d` entities with `TextSpan` components will act as additional text sections.
```